site/app/views/forum/ForumThreadView.php

```diff
@@ -1199,7 +1182,7 @@ public function createPost(array $thread, array $post, $unviewed_posts, $first,
             }
         }
 
-        if ($this->core->getUser()->getGroup() == 4) {
```

Comment:
This is the source of your PHP lint failure: you fixed a loose comparison so PHPstan complains that not enough loose comparisons were found. To fix this, [rebuild the baseline](https://submitty.org/developer/testing/linting_static_analysis).